OpenEdXMobile/src/main/java/org/edx/mobile/deeplink/BranchLinkManager.kt

```diff
@@ -0,0 +1,18 @@
+package org.edx.mobile.deeplink
+
+import android.content.Context
+import org.edx.mobile.logger.Logger
+import org.json.JSONObject
+
+object BranchLinkManager {
+    val logger: Logger = Logger(this.javaClass)
+    const val KEY_CLICKED_BRANCH_LINK = "+clicked_branch_link"
+
+    fun didReceiveLink(context: Context, paramsJson: JSONObject) {
```

Comment:
IMO `checkAndReactBranchLink` is more appropriate.

Reply:
done